src/app/api/following/route.ts

```diff
@@ -18,11 +18,11 @@ export async function GET(req: NextRequest) {
         .from("following")
         .select("following")
         .eq("follower", username));
-
-      // If no user was found, throw an error
-      if (data && data.length === 0) {
-        throw new Error(`User with username ${username} not found`);
-      }
+      //pretty sure I don't need this..
```

Comment:
Yeah, since the following functionality on the frontend will only allow users to "follow" existing users, this won't hit. Good to make sure about this first lol, you can eventually get rid of the commented out portions.

Reply:
Perfect, will clean this up as well too, thank you.